scripts/list-releases.sh

```diff
@@ -6,15 +6,12 @@ if [ "$1" = "" ]; then
     exit 1
 fi
 
-opsctl list installations > list-of-installations.txt
-gsctl list endpoints > list-of-endpoints.txt
 
 provider=$1
 active_release=$2
 for installation in $(opsctl list installations --provider=$provider --short); do
-		gsctl select endpoint "${inallation_data[0]}"
-		
-		if [ "$active_release" = "" ]
+		gsctl select endpoint "${installation[0]}"
```

Comment:
why do you bother doing this? You can run `gsctl list releases -e <endpoint>` without selecting it